sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/ContainerRegistryClient.java

```diff
@@ -73,49 +68,54 @@ public String getEndpoint() {
-        return this.deleteRepositoryWithResponse(repositoryName, Context.NONE).getValue();
+    public void deleteRepository(String repositoryName) {
+        this.deleteRepositoryWithResponse(repositoryName, Context.NONE);
     }
 
     /**
      * Delete the repository identified by 'repositoryName'.
      *
      * @param repositoryName Name of the repository (including the namespace).
      * @param context The context to associate with this operation.
-     * @return deleted repository properties.
+     * @return Completion response.
      * @throws ClientAuthenticationException thrown if the client's credentials do not have access to modify the namespace.
-     * @throws ResourceNotFoundException thrown if the repository to be deleted does not exist.
-     * @throws NullPointerException thrown if the name is null.
+     * @throws NullPointerException thrown if the 'repositoryName' is null.
+     * @throws IllegalArgumentException thrown if the 'repositoryName' is empty.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Response<DeleteRepositoryResult> deleteRepositoryWithResponse(String repositoryName, Context context) {
+    public Response<Void> deleteRepositoryWithResponse(String repositoryName, Context context) {
         return this.asyncClient.deleteRepositoryWithResponse(repositoryName, context).block();
     }
 
     /**
-     * Get an instance of repository client from the registry client.
+     * Creates a new instance of {@link ContainerRepository} object for the specified repository.
      *
-     * @param repositoryName Name of the repository (including the namespace).
-     * @return repository client.
+     * @param repositoryName Name of the repository to reference.
+     * @return A new {@link ContainerRepository} for the desired repository.
+     * @throws NullPointerException if 'repositoryName' is null.
+     * @throws IllegalArgumentException if 'repositoryName' is empty.
      */
     public ContainerRepository getRepository(String repositoryName) {
         return new ContainerRepository(this.asyncClient.getRepository(repositoryName));
     }
 
     /**
-     * Get an instance of registry artifact class.
+     * Creates a new instance of {@link RegistryArtifact} object for the specified artifact.
      *
-     * @param repositoryName Name of the repository (including the namespace).
-     * @param tagOrDigest Tag or digest associated with the artifact.
-     * @return repository client.
+     * @param repositoryName Name of the repository to reference.
+     * @param digest Either a tag or digest that uniquely identifies the artifact.
+     * @return A new {@link RegistryArtifact} object for the desired repository.
+     * @throws NullPointerException if 'repositoryName' is null.
+     * @throws IllegalArgumentException if 'repositoryName' is empty.
+     * @throws NullPointerException if 'digest' is null.
+     * @throws IllegalArgumentException if 'digest' is empty.
```

Comment:
Don't need to repeat the exceptions here. Define the exception once and describe all cases which can throw that exception.